source/includes/_stackpath.md

```diff
@@ -0,0 +1,3 @@
+# Stackpath plugin
+
+The Stackpath plugin provides endpoints for carrying out operations on CloudMC Edge computing entities. It also enforces CloudMC's environment-centric multi-tenancy model and RBAC over top of Stackpath.
```

Comment:
Is it:

1) `It also enforces CloudMC's environment-centric multi-tenancy model and RBAC over top of Stackpath.`
or
2) `It also enforces CloudMC's environment-centric, multi-tenancy model and RBAC over top of Stackpath.`

Reply:
Checked with GCP, Azure, and Swift. In there it is as it is in #1.